### ISSUE TITLE
The main purpose of the function: get_label

### ISSUE THREAD

**Issue by vhientran** [wzhouad/ATLOP#16]
Hi @wzhouad ,

Thanks so much for releasing your source code. I only wonder about the main purpose of the function get_label() in the file losses.py in calculating the final loss. Could you please explain it? Thanks for your help!

**Comment by wzhouad** [comment]
Hi,

The function is for transforming class probabilities to predicted labels. It is used only in inference.

**Comment by vhientran** [comment]
Hi @wzhouad ,

I got it. Thank you very much!

**Comment by vhientran** [comment]
Hi @wzhouad ,

Sorry for disturbing you, but I just have one more question. In the training phase, the parameters of the pre-trained BERT will be fine-tuned or fixed? Thank you!

**Comment by wzhouad** [comment]
They are finetuned.

**Comment by vhientran** [comment]
Many thanks!
